.github/workflows/ci-matrix.yml

```diff
@@ -43,8 +43,19 @@ jobs:
           rustup check
           rustup component add rustfmt
           rustup component add clippy
-          ./ci/all.sh
-          ./ci/multinode_integration_test.sh || echo "multinode integration tests failed"
+          if ! ./ci/all.sh; then
```

Comment:
Revert these changes please. 

But this line is also wrong:
`./ci/multinode_integration_test.sh || echo "multinode integration tests failed"`
It should be like` ./ci/multinode_integration_test.sh` alone.